code/pcgen-launch4j.cfg.xml

```diff
@@ -1,23 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <launch4jConfig>
   <dontWrapJar>true</dontWrapJar>
   <headerType>gui</headerType>
   <jar>pcgen.jar</jar>
-  <outfile>C:\Projects\pcgendev\code\pcgen.exe</outfile>
+  <outfile>Z:\Netbeans\pcgen\code\pcgen.exe</outfile>
```

Comment:
This looks like a custom build change. Should this be using 'Netbeans' when the release monkeys use different IDEs?